crate/cli/src/actions/kms/secret_data/revoke_secret.rs

```diff
@@ -0,0 +1,38 @@
+use clap::Parser;
+use cosmian_kms_client::{KmsClient, kmip_2_1::kmip_types::UniqueIdentifier};
+
+use crate::{
+    actions::kms::{
+        labels::KEY_ID,
+        shared::{get_key_uid, utils::revoke},
+    },
+    error::result::KmsCliResult,
+};
+
+/// Revoke a secret data
+///
+/// When a secret data key is revoked, it can only be exported by the owner of the key,
```

Comment:
When a secret data <s>key</s> is revoked, it can only be exported by the owner of the <s>key</s>secret data